examples/error_message.rs

```diff
@@ -6,7 +6,7 @@ fn main() {
     let access_token = std::env::var("ROLLBAR_ACCESS_TOKEN").unwrap_or("".to_string());
     let environment = std::env::var("ROLLBAR_ENVIRONMENT").unwrap_or("".to_string());
     let client = rollbar::Client::new(access_token, environment);
-    let _ = report_error_message!(client, "＿|￣|○").join();
+    let _ = report_error_message!(client, "＿|￣|○");
```

Comment:
send() no longer returns a join handle, it blocks on the async itself so no join or block or anything needed here.

Reply:
we don't use `join` anywhere in our code afaict, i'm guessing either our code is non-blocking and we let the thread live forever or we weren't using the lib correctly. maybe it's both and we're allowing threads to spawn that never join?

Reply:
it seems bad to stall the cpu for this request to go out basically halting service on that thread for that time, it also seems bad to have orphans if that is happening. i'm also wondering what happens if we're getting way too many panics and hit thread limits.

possibly a better architecture is a single thread and a queue with some buffer size serving all panics, it drops if it's full. thoughts?